### ISSUE TITLE
Assessing significance of genes

### ISSUE THREAD

**Issue by jcgrenier** [GabrielHoffman/variancePartition#67]
Hello @GabrielHoffman ,

I'm used to normally use the corrected p-values or FDR values in order to assess for gene significance. How would you interpret the results and select the significant hits in this case if we can't rely on corrected p-values? How to make sure it's not a false positive if it's not corrected for multiple testing?

Thanks a lot for your help.

JC

**Comment by GabrielHoffman** [comment]
Hi, 
Can you clarify your concern that "we can't rely on corrected p-values?".  The `dream()` method controls the false positive rate, so the FDRs are correctly calibrated.

Gabriel

**Comment by jcgrenier** [comment]
Hello,
I mean that in your documentation, you are referring to the z.std score to make downstream analysis. I see however that it's the t-statistics that are not directly comparable. So, should I use a threshold on the z.std to select genes that are differentially expressed or should I refer to the p-value or the adjusted p-value?

Thanks a lot for your advice.

JC

**Comment by GabrielHoffman** [comment]
In standard `limma` analysis, all the t-statistics have the same degrees of freedom.  So the rank of t-statistics, p-values (and adjusted p-values) is the same.

But the linear mixed model used by `dream()` is more complicated: the degrees of freedom of the null distribution is different for each gene and depends on the data.  So the t-statistic along is not sufficient to get the p-value.  In the back end `topTable()` uses the t-statistics and the gene-specific degrees of freedom value to compute the p-value.  This is then used to compute the adjusted p-value.  

In summary, you can use the adjusted p-value just like you expected

Separately, sometimes users want a single statistic that is comparable across genes like in the original `limma`.  So I created the `z.std` column that converts the t-statistic and gene-specific degrees of freedom to a z-statistical that is comparable across all genes.  This is used for [zenith for gene set testing](https://gabrielhoffman.github.io/zenith/index.html), but not much else.

Gabriel

**Comment by jcgrenier** [comment]
Hello Gabriel,

Thanks for that pretty clear answer! 

Have a nice week end!
JC
